### ISSUE TITLE
[APP SUBMITTED]: 403 Client Error: Forbidden for url: https://trakt.tv/oauth/token

### ISSUE THREAD

**Issue by E-Raw** [SickRage/sickrage-issues#486]
### INFO
Python Version: **2.7.11 (default, Dec 17 2015, 15:58:38) [GCC 4.7.3 20130102 (prerelease)]**
Operating System: **Linux-3.10.35-x86_64-with-glibc2.2.5**
Locale: UTF-8
Branch: **master**
Commit: SickRage/SickRage@93536ada05ad3fa9cfb4b18f61787bc8e441f45f
Link to Log: https://gist.github.com/c30d1b288f742019934b
### ERROR
```
Thread-58 :: Could not connect to Trakt. Code error: 403 [93536ad]
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by fernandog** [comment]
@E-Raw check your trakt user/pass and api SR settings

**Comment by duramato** [comment]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-43-generic-i686-with-Ubuntu-14.04-trusty**
Locale: None
Branch: **smack-you**
Commit: SickRage/SickRage@1c63e6405c3d1b17cfe2e3852ff38efe1f817e06
Link to Log: https://gist.github.com/5eb12cfbcf0fc3785151
### ERROR
```
TORNADO :: API :: error 4 [1c63e64]
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by E-Raw** [comment]
Hi Fernando,

I've tried both user login and email address as ID then a fresh PIN generated from Trakt.
So I presume it is more related to the interface involved in this scenario as there is no password in the process, only the PIN.

As it runs on a Syno, which is connected to a VPN, I've made a *curl http://trakt.tv* and it divert it to CloudFlare.

I'll try without VPN connected to see the result but I think it will do the trick.

Best regards and thank you for the great work you're doing here ;-)
